check why swarm replicas failed randomly

check why the swarm.bats test is failing randomly

Thanks for opening this issue @GabyCT. 

Some logs from Jenkins:

```
not ok 4 check mtu values in different interfaces
# (in test file integration/swarm/swarm.bats, line 100)
#   `[ "$mtu_value_eth0" = "$mtu_value_tap0" ]' failed
# Swarm initialized: current node (ecncxlgj6lra727qykus0r9y6) is now a manager.
# 
# To add a worker to this swarm, run the following command:
# 
#     docker swarm join --token SWMTKN-1-344w4asinwwknzv7q6qu8l6oozajrqvhf5yyghfokeg6vr28xq-b8h5dqf0678w85jboco95moaq 10.0.0.5:2377
# 
# To add a manager to this swarm, run 'docker swarm join-token manager' and follow the instructions.
# 
# okl2bbjfdytwdvq49y3nt4nkw
# Since --detach=false was not specified, tasks will be created in the background.
# In a future release, --detach=false will become the default.
# ID                  NAME                MODE                REPLICAS            IMAGE                 PORTS
# okl2bbjfdytw        testswarm           replicated          0/4                 gabyct/nginx:latest   *:8080->80/tcp
# ID                  NAME                MODE                REPLICAS            IMAGE                 PORTS
# okl2bbjfdytw        testswarm           replicated          0/4                 gabyct/nginx:latest   *:8080->80/tcp
# ID                  NAME                MODE                REPLICAS            IMAGE                 PORTS
# okl2bbjfdytw        testswarm           replicated          0/4                 gabyct/nginx:latest   *:8080->80/tcp
# ID                  NAME                MODE                REPLICAS            IMAGE                 PORTS
# okl2bbjfdytw        testswarm           replicated          0/4                 gabyct/nginx:latest   *:8080->80/tcp
# ID                  NAME                MODE                REPLICAS            IMAGE                 PORTS
# okl2bbjfdytw        testswarm           replicated          0/4                 gabyct/nginx:latest   *:8080->80/tcp
# ID                  NAME                MODE                REPLICAS            IMAGE                 PORTS
# okl2bbjfdytw        testswarm           replicated          0/4                 gabyct/nginx:latest   *:8080->80/tcp
# ID                  NAME                MODE                REPLICAS            IMAGE                 PORTS
# okl2bbjfdytw        testswarm           replicated          0/4                 gabyct/nginx:latest   *:8080->80/tcp
# ID                  NAME                MODE                REPLICAS            IMAGE                 PORTS
# okl2bbjfdytw        testswarm           replicated          0/4                 gabyct/nginx:latest   *:8080->80/tcp
# ID                  NAME                MODE                REPLICAS            IMAGE                 PORTS
# okl2bbjfdytw        testswarm           replicated          0/4                 gabyct/nginx:latest   *:8080->80/tcp
# ID                  NAME                MODE                REPLICAS            IMAGE                 PORTS
# okl2bbjfdytw        testswarm           replicated          0/4                 gabyct/nginx:latest   *:8080->80/tcp
# testswarm
# Node left the swarm.
Makefile:49: recipe for target 'swarm' failed
make: *** [swarm] Error 1
Build step 'Execute shell' marked build as failure
```

--- a/integration/swarm/swarm.bats
+++ b/integration/swarm/swarm.bats
@@ -64,6 +64,7 @@ setup() {
 }
 
 @test "check_service_ip_among_the_replicas" {
+	skip "This is not working (https://github.com/clearcontainers/tests/issues/694)"
 	service_name=$($DOCKER_EXE service ls --filter name="${SERVICE_NAME}" -q)
 	ip_service=$($DOCKER_EXE service inspect $SERVICE_NAME \
 		--format='{{range .Endpoint.VirtualIPs}}{{.Addr}}{{end}}' | cut -d'/' -f1)

